python/sdist/amici/de_model.py

```diff
@@ -2540,6 +2543,176 @@ def _process_heavisides(
 
         return dxdt
 
+    @property
+    def _components(self) -> list[ModelQuantity]:
+        """
+        Returns the components of the model
+
+        :return:
+            components of the model
+        """
+        return (
+            self._algebraic_states
+            + self._algebraic_equations
+            + self._conservation_laws
+            + self._constants
+            + self._differential_states
+            + self._event_observables
+            + self._events
+            + self._expressions
+            + self._log_likelihood_ys
+            + self._log_likelihood_zs
+            + self._log_likelihood_rzs
+            + self._observables
+            + self._parameters
+            + self._sigma_ys
+            + self._sigma_zs
+            + self._splines
+        )
+
+    def _process_hybridization(self, hybridization: dict) -> None:
+        """
+        Parses the hybridisation information and updates the model accordingly
+
+        :param hybridization:
+            hybridization information
```

Comment:
This could be more informative :sweat_smile:
Is this documented in petab-sciml? Add reference?